packages/EasyRandom/src/Bridge/Symfony/Resources/config/services.yaml

```diff
@@ -3,5 +3,14 @@ services:
         autowire: true
```

Comment:
Can you also add `private: true` so this way we're sure that the services are private by default so they will be removed from the container if not used.